doc/tutorial/c_tutorial/c_tutorial.rst

```diff
@@ -0,0 +1,487 @@
+************************
+Integrating C with DDlog
+************************
+
+.. contents::
+
+Prerequisites
+=============
+
+First, you need to obtain DDlog_ from the official repository, which contains instructions how to install and set it up.
+There is also an expectation that you are familiar with DDlog syntax and commands, i.e., you have read the tutorial or at least skimmed through it.
+
+.. _DDlog: https://github.com/vmware/differential-datalog
+
+Introduction
+============
+
+In this set of tutorials we are going to explore various features of DDlog programming language and learn how to integrate DDlog program into C program.
+To do that we are going to build a simple yet functional application as it is more fun to build real programs.
+We will start by writing a simple program with a limited functionality to get familiar with the basics of DDlog.
+Afterwards, over the course of this set of tutorials, we will gradually improve our application by applying more advanced techniques offered by DDlog.
+
+Tutorial #1 - Reachability Monitor
+==================================
+
+In our first tutorial we are going to build a program that will calculate the reachability information in the network.
+User will update link status between nodes through CLI and our program will calculate the rest.
+The purpose of this program is to provide a simple interface to a network operator and return information regarding which node can reach which.
+Thus, the only concern of network operator is to update link status without having to worry about doing those calculations themself.
+We are going to test our program with the network consisting of four nodes: Menlo Park, Santa Barbara, Los Angeles, and Salt Lake City.

```

Comment:
I would move these two lines a little lower, when you actually show how to insert the data.